### ISSUE TITLE
GoHTML5 Templates

### ISSUE THREAD

**Issue by Claire-Crawford** [Claire-Crawford/myWebBiz#2]
- [ ] function for nav bar 

- [ ] function for footer

- [ ] function for header 

- [ ] function for CTA

**Comment by comaldave** [comment]
VueJS is deprecated. Please advise when any of the above items are completed and ready for templating. They are ready for templating when you have completed the styling using Bootstrap 5. If for example, you have completed the footer element in Bootstrap 5 then create a new issue assigning me to create the GoHtml5 Template for the footer. Once you see how it is done, you shall have no difficulties with the other templates.
